### PR TITLE
switch all repo URIs to https

### PR DESCRIPTION
**Pull request by costanic**
this should alleviate ssh key issues when using the docker
container for builds.

### CODE REVIEW

**Review by ryannowarm — APPROVED** [review]

**Review by CristianPrundeanuARM — APPROVED** [review]

**Review by mray190 — APPROVED** [review]

**Comment by mray190** [comment]
Will merge after CI passes

**Review by moonchen — APPROVED** [review]

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -304,7 +304,7 @@ parts:
         bin/edge-proxy: wigwag/system/bin/edge-proxy
     devicedb:
       plugin: go
-      source: git@github.com:armPelionEdge/devicedb.git
```
Comment (JanneKiiskila) [inline snap/snapcraft.yaml:307]: Great, you've also changed the domain to PelionIoT! Good!

**Comment by JanneKiiskila** [comment]
While we are at it, let's get rid of the armmbed armPelionEdge domains completely. These are still there:

````
(repo-venv) jankii01@ubuntu1804:~/mbed/snap-pelion-edge$ git grep -i "armpelion"
EdgeJenkinsfile.groovy:// Function is implemented in https://github.com/armPelionEdge/edge-ci/blob/master/vars/snapPelionEdge.groovy
README.rpi3.md:    git clone https://github.com/armpelionedge/snap-pelion-edge
snap/snapcraft.yaml:      go-importpath: github.com/armPelionEdge/devicedb
snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/edge-node-modules.git
snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/pe-utils.git
snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/maestro-shell.git
snap/snapcraft.yaml:        install -d go/src/github.com/armPelionEdge
snap/snapcraft.yaml:        cd go/src/github.com/armPelionEdge
snap/snapcraft.yaml:        git clone https://github.com/armPelionEdge/maestro-shell.git
snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro-shell/
snap/snapcraft.yaml:        go build github.com/armPelionEdge/maestro-shell
snap/snapcraft.yaml:        go install github.com/armPelionEdge/maestro-shell
snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/maestro.git
snap/snapcraft.yaml:        install -d go/src/github.com/armPelionEdge
snap/snapcraft.yaml:        cd go/src/github.com/armPelionEdge
snap/snapcraft.yaml:        git clone https://github.com/armPelionEdge/maestro.git
snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro/
snap/snapcraft.yaml:        go install github.com/armPelionEdge/maestro
snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/
snap/snapcraft.yaml:        install ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/grease_echo ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/grease_echo
snap/snapcraft.yaml:        install ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/standalone_test_logsink ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/standalone_test_logsink
snap/snapcraft.yaml:        LIBS_DIR="${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/lib"
```

Let's get them sorted out on the same go, please @costanic .

**Comment by costanic** [comment]
> While we are at it, let's get rid of the armmbed armPelionEdge domains completely. These are still there:
> 
> ```
> (repo-venv) jankii01@ubuntu1804:~/mbed/snap-pelion-edge$ git grep -i "armpelion"
> EdgeJenkinsfile.groovy:// Function is implemented in https://github.com/armPelionEdge/edge-ci/blob/master/vars/snapPelionEdge.groovy
> README.rpi3.md:    git clone https://github.com/armpelionedge/snap-pelion-edge
> snap/snapcraft.yaml:      go-importpath: github.com/armPelionEdge/devicedb
> snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/edge-node-modules.git
> snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/pe-utils.git
> snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/maestro-shell.git
> snap/snapcraft.yaml:        install -d go/src/github.com/armPelionEdge
> snap/snapcraft.yaml:        cd go/src/github.com/armPelionEdge
> snap/snapcraft.yaml:        git clone https://github.com/armPelionEdge/maestro-shell.git
> snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro-shell/
> snap/snapcraft.yaml:        go build github.com/armPelionEdge/maestro-shell
> snap/snapcraft.yaml:        go install github.com/armPelionEdge/maestro-shell
> snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/maestro.git
> snap/snapcraft.yaml:        install -d go/src/github.com/armPelionEdge
> snap/snapcraft.yaml:        cd go/src/github.com/armPelionEdge
> snap/snapcraft.yaml:        git clone https://github.com/armPelionEdge/maestro.git
> snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro/
> snap/snapcraft.yaml:        go install github.com/armPelionEdge/maestro
> snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/
> snap/snapcraft.yaml:        install ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/grease_echo ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/grease_echo
> snap/snapcraft.yaml:        install ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/standalone_test_logsink ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/standalone_test_logsink
> snap/snapcraft.yaml:        LIBS_DIR="${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/lib"
> ```
> 
> Let's get them sorted out on the same go, please @costanic .

@JanneKiiskila I can likely address a few of these, but unfortunately due to the nature of the Go programming language I cannot address all of them without extensive changes to the underlying components.

**Comment by costanic** [comment]
> > While we are at it, let's get rid of the armmbed armPelionEdge domains completely. These are still there:
> > ```
> > (repo-venv) jankii01@ubuntu1804:~/mbed/snap-pelion-edge$ git grep -i "armpelion"
> > EdgeJenkinsfile.groovy:// Function is implemented in https://github.com/armPelionEdge/edge-ci/blob/master/vars/snapPelionEdge.groovy
> > README.rpi3.md:    git clone https://github.com/armpelionedge/snap-pelion-edge
> > snap/snapcraft.yaml:      go-importpath: github.com/armPelionEdge/devicedb
> > snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/edge-node-modules.git
> > snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/pe-utils.git
> > snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/maestro-shell.git
> > snap/snapcraft.yaml:        install -d go/src/github.com/armPelionEdge
> > snap/snapcraft.yaml:        cd go/src/github.com/armPelionEdge
> > snap/snapcraft.yaml:        git clone https://github.com/armPelionEdge/maestro-shell.git
> > snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro-shell/
> > snap/snapcraft.yaml:        go build github.com/armPelionEdge/maestro-shell
> > snap/snapcraft.yaml:        go install github.com/armPelionEdge/maestro-shell
> > snap/snapcraft.yaml:      source: https://github.com/armPelionEdge/maestro.git
> > snap/snapcraft.yaml:        install -d go/src/github.com/armPelionEdge
> > snap/snapcraft.yaml:        cd go/src/github.com/armPelionEdge
> > snap/snapcraft.yaml:        git clone https://github.com/armPelionEdge/maestro.git
> > snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro/
> > snap/snapcraft.yaml:        go install github.com/armPelionEdge/maestro
> > snap/snapcraft.yaml:        cd ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/
> > snap/snapcraft.yaml:        install ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/grease_echo ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/grease_echo
> > snap/snapcraft.yaml:        install ${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/standalone_test_logsink ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/standalone_test_logsink
> > snap/snapcraft.yaml:        LIBS_DIR="${GOPATH}/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/lib"
> > ```
> > 
> > 
> > Let's get them sorted out on the same go, please @costanic .
> 
> @JanneKiiskila I can likely address a few of these, but unfortunately due to the nature of the Go programming language I cannot address all of them without extensive changes to the underlying components.

@JanneKiiskila On second thought, I'll do that in another PR because we have other efforts to do just that.  I'd like to keep this PR focused on just changing ssh to https.

**Comment by costanic** [comment]
rebased on dev branch to force CI to kick off again...

**Pull request merged**